cde-pentaho5/resource/resources/custom/amd-components/Dashboard/DashboardComponent.js

```diff
@@ -78,9 +78,20 @@ define([
             if(myself.isParameterPublic(otherParam)) {
               var eventName = myParam + ":fireChange";
               var fun = function(evt) {
-                reqDash.fireChange(otherParam, evt.value);
+                if(reqDash.getParameterValue(otherParam) !== evt.value) {
+                  reqDash.fireChange(otherParam, evt.value);
+                }
               };
               myself.dashboard.on(eventName, fun);
+              var subDashboardEventName = otherParam + ":fireChange";
+              var subDashboardfun = function (evt) {
```

Comment:
I'd like this to be consistent with the name chosen above, so `reqDashEventName` and `reqDashFun`